app/src/main/java/chat/rocket/android/authentication/signup/presentation/SignupPresenter.kt

```diff
@@ -66,6 +67,10 @@ class SignupPresenter @Inject constructor(private val view: SignupView,
                         localRepository.save(LocalRepository.CURRENT_USERNAME_KEY, me.username)
                         saveAccount(me)
                         registerPushToken()
+                        var loginCredentials: Credential = Credential.Builder(email)
```

Comment:
Not a problem, but are you really needing to declare the `loginCredentials` type (`Credential`)?  If no, you don't need to use that (it is implicit when seeing how the variable is build).

More important:

Avoid using `var` when it is not need. Change for `val` please (variable is never modified!)!